external/src/commonMain/kotlin/Interop.kt

```diff
@@ -0,0 +1,73 @@
+package com.juul.indexeddb.external
+
+@PublishedApi
+internal expect fun JsAny.unsafeCast(): IDBKey
+
+@PublishedApi
+internal expect fun Int.unsafeCast(): IDBKey
+
+@PublishedApi
+internal expect fun Double.unsafeCast(): IDBKey
+
+@PublishedApi
+internal expect fun String.unsafeCast(): IDBKey
+
+@PublishedApi
+internal expect fun Array<IDBKey>.unsafeCast(): IDBKey
+
+public expect interface JsAny
```

Comment:
Would love to see a doc comment on this explaining it. From my understanding, `JsAny` is meant as a marker of DB-safe types. Ideally something that includes the`external interface MyRowType : JsAny` use case.

Reply:
`JsAny` is really just there so that there can be common code for Js and Wasm. Wasm requires externals to extend `JsAny`, so that had to be represented in the common code. 

I believe Kotlin is working on improving the interop story for Js and Wasm, so this can likely get simplified in the future.